docs/upload-your-snap.md

```diff
@@ -1,69 +1,154 @@
-# Upload to the store
+---
+title: "Release your snap"
+---
 
-So you've been working hard on your snap, and you finally have it to the point
-where you're ready to share it with the world? Great! You can use Snapcraft to
-upload it directly to the store.
 
-## The grade option
+In order to share your snaps with the world, you need to publish them in a
+store. This page describes the publication process you will follow for the
+Ubuntu-hosted Store.
 
-`grade` in your `snapcraft.yaml` file can be either `stable` or `devel`. It
-essentially tags the resulting snap for life with the grade it should have.
+Although, snaps are not tied to a specific type of store and you can host
+them any way you want, see the [Store](/docs/core/store) page for more
+details.
 
-How is this useful? Imagine you are developing a snap and have your continuous
-integration setup to always push to the `edge` channel; in this case one would
-set `grade: devel`. This ensures that this snap is not accidentally published
-to the `stable` channels.
+## 1. Create a store account
 
-A snap with `grade: devel` cannot be released to the `stable` or `candidate`
-channels.
+To publish snaps to the Ubuntu Store you will need to create an account
+on [myapps.developer.ubuntu.com](https://myapps.developer.ubuntu.com/). This
+is your developer portal where you can customize how your snaps are presented,
+review your release, and control publishing.
 
+You'll need to choose a unique "developer namespace" as part of the account
+creation process. This name will be visible by users and associated with your
+published snaps.
 
-## Build the snap
+Once you've confirmed your account, you're ready to start pushing your snaps
+to the Store.
 
-Get into the directory containing the `snapcraft.yaml` file, and do the following:
+Make sure the `snapcraft` and `snap` commands know about you by logging in
+using the email address attached to your account.
 
-    $ snapcraft snap
-    [...]
-    Snapped foo_1_amd64.snap
 
+### Example
 
-## Authenticate to the store
+    $ snapcraft login
+    $ snap login you@yourdomain.com
 
-First of all, you need to give Snapcraft permission to communicate with the
-store on your behalf:
+Note that `logout` commands are available as well.
```

Comment:
It's just one. So it should be: Note that the `logout` command is available as well.